sdk/appcenter/src/main/java/com/microsoft/appcenter/persistence/DatabasePersistence.java

```diff
@@ -592,15 +611,15 @@ public void close() {
         mDatabaseManager.close();
     }
 
-    private List<Long> getCorruptedIds(SQLiteQueryBuilder builder, String[] selectionArgs) {
+    private List<Long> getLogsIds(SQLiteQueryBuilder builder, String[] selectionArgs) {
```

Comment:
What is the purpose of renaming this?

Reply:
It was required for implementing deleting logs by time (was removed after feedback). This method doesn't do anything special for "corrupted" entries: it just returns only ids.